### DISCUSSION TITLE
How do you create new sources?

### DISCUSSION THREAD

**Discussion by jpdrennon** [kinkofer/FightClub5eXML#189] [Q&A] [answered]
This is more of a general question for the folks that have put in a TON of work on this. How do you go about creating an entire new source xml? Do you painstakingly manually create each item, background, race, etc? Or do you run a script that converts the PDF version of a new book into xml? Or some other method? This is really just a question being asked as someone who would like to contribute in the future and not just take what you all have built. Either way, much appreciated work you've done.

**Comment by kinkofer** [comment, marked as answer]
The short answer is there’s another larger community converting source data to json, I don’t know how they do it, probably a mixture of automated and manual. I work with a smaller community to convert the json to xml.

Details are intentionally left fuzzy to protect everyone involved.

**Reply by jpdrennon** [reply]
Thanks for that answer. Just know that your work is appreciated.
